edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-batching.adoc

```diff
@@ -0,0 +1,60 @@
+= EDIFACT EDI Connector 2.3 - Batch Outbound EDIFACT Messages - Mule 4
+
+For EDI messages that are not time-sensitive, such as invoice and payment remittances, you can use the Batch Write operation to merge multiple individual messages into a single interchange to send to your trading partners.
+
+== Batching Stages
+
+Use the Anypoint EDIFACT EDI Connector Batch Write operation to merge individual EDIFACT messages into a single transmission to send to your trading partners.
```

Comment:
GLOBAL re. connector names: The official name of each is "Anypoint Connector for...," with the abbreviation being "... Connector." In Connector Land, we don't slap "Anypoint" alone in front of the product name.

Reply:
```suggestion
Use the Anypoint Connector for EDIFACT EDI (EDIFACT EDI Connector) Batch Write operation to merge individual EDIFACT messages into a single transmission to send to your trading partners.
```